web/themes/custom/novel/templates/views/node--list-teaser.html.twig

```diff
@@ -0,0 +1,40 @@
+{# Check if the event image field has items and set variables for the media entity and image file #}
+{% if content.field_event_image['#items'] is not empty %}
+  {% set media_entity = content.field_event_image['#items'].0.entity %}
+    {% set image_field = media_entity.field_media_image %}
+    {% set image_file = image_field.entity %}
+{% endif %}
+
+<a href="{{ content.event_link }}" class="event-list-item arrow__hover--right-small">
+  <div class="event-list-item__image-container">
+    {% if image_file %}
+      <img src="{{ file_url(image_file.fileuri) }}" alt="{{ image_field.alt }}" class="event-list-item__image" />
+    {% endif %}
+  </div>
+  <div class="event-list-item__content">
+    <div class="tag tag--fill cursor-pointer tag--small event-list-item__tag">foredrag</div>
+    <time class="event-list-item__date" datetime="{{ content.field_event_date['#items'].0.value|date('Y-m-d') }}">
```

Comment:
Please reconsider using a `<time>` tag here.

When I think about it this tag may contain a range. As I understand it the current logic will not work here.